### PR TITLE
Add jadx

### PR DESCRIPTION
**Pull request by naacbin**
Add of jadx, a tool to decompile Dalvik bytecode from apk.

Code : https://github.com/skylot/jadx

### CODE REVIEW

File: packages/jadx.vm/tools/chocolateyinstall.ps1 [hunk 1]
```diff
@@ -0,0 +1,10 @@
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+
+$toolName = 'jadx'
+$category = 'Forensic'
```
Comment (Ana06) [inline packages/jadx.vm/tools/chocolateyinstall.ps1:5]:
```suggestion
$category = 'Java & Android'
```

**Review by Ana06 — COMMENTED** [review]
Is this tool similar to apktool and Bytecode Viewer? Do we need all the tools?

**Comment by naacbin** [comment]
It's not like apktool but seems similar to Bytecode Viewer (both have a GUI).
However, I'm not familiar with Bytecode Viewer and I use jadx a lot. Incorporating this tool into the packages is still important, even if it duplicates functionality with another tool. It's similar to Ghidra/IDA, question of personal preferences.

File: packages/jadx.vm/tools/chocolateyuninstall.ps1 [hunk 2]
```diff
@@ -0,0 +1,7 @@
+$ErrorActionPreference = 'Continue'
+Import-Module vm.common -Force -DisableNameChecking
+
+$toolName = 'jadx'
+$category = 'Forensic'
```
Comment (Ana06) [inline packages/jadx.vm/tools/chocolateyuninstall.ps1:5]:
```suggestion
$category = 'Java & Android'
```

**Review by Ana06 — CHANGES_REQUESTED** [review]

**Comment by Ana06** [comment]
@mandiant/flare-vm opinions on adding this tool to the default FLARE-VM configuration?

**Comment by mr-tz** [comment]
bytecodeviewer includes jadx so I think this is a duplicate

**Comment by naacbin** [comment]
Indeed, the decompiler is included in bytecodeviewer. I still prefer the GUI of JADX but bytecodeviewer seems more powerful. I leave it to you to decide whether to close it or not.

**Comment by Ana06** [comment]
Thanks for the input @mr-tz! I vote for closing.

**Pull request closed without merging**